mlrun/db/base.py

```diff
@@ -738,6 +738,7 @@ def list_model_endpoints(
         top_level: bool = False,
         uids: Optional[list[str]] = None,
         latest_only: bool = False,
+        metrics: Optional[list[str]] = None,
```

Comment:
maybe instead of managing 2 different params, we can have only `tsdb_metrics` that for example can be either a list or a str. if it's `*` we can return all tsdb metrics, otherwise we return the provided metrics in the list (or a single metric if the user provides a str). If it's none (by default) we don't return any tsdb metrics.